src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -625,6 +626,14 @@ class GenericLogStruct(NamedTuple):
   pass
 
 
+class CommonLogStruct(NamedTuple):
```

Comment:
What do you think of doing set_common_log_context in the first call to a log function, so it's impossible to forget to do.

Reply:
I think this suggested approach will work much better with multiprocessing code since we typically use spawn.